src/extension/background-script/index.ts

```diff
@@ -154,13 +161,23 @@ async function init() {
 }
 
 console.info("Welcome to Alby");
-init().then(() => {
-  if (isFirstInstalled && !state.getState().getAccount()) {
-    utils.openUrl("welcome.html");
-  }
-  if (isRecentlyUpdated) {
-    migrate();
-  }
-});
+init()
+  .then(() => {
+    if (resolveInit) {
+      resolveInit();
+    }
+    if (isFirstInstalled && !state.getState().getAccount()) {
+      utils.openUrl("welcome.html");
+    }
+    if (isRecentlyUpdated) {
+      migrate();
```

Comment:
Should we consider moving migrate() to the bottom of the init() function?

Reply:
You probably mean to the top of the init function, right? For me it would make sense to await the migration and only then `resolveInit` to avoid calls being executed while migrations are still taking place.

Reply:
I agree `resolveInit` shouldn't be called until the migrations have run. But don't the migrations depend on `init`? that's why I suggested it being moved to the last line in the init() function.